src/Components/Bookpage/BookDetails/Reviews/index.js

```diff
@@ -0,0 +1,13 @@
+import "./reviews.css"
+
+function Reviews({ review }) {
```

Comment:
Rename to 'Review' since this is singular

Reply:
done